parlai/agents/alice/alice.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Agent uses the Free Alice AIML interpreter to generate replies to observations.
+"""
+
+import os
+from parlai.core.agents import Agent
+
+try:
+    import aiml
+except ImportError:
+    raise ImportError(
+        (
+            "ALICE agent needs python-aiml installed. Please run:\n "
+            "pip install git+https://github.com/paulovn/python-aiml.git"
+        )
+    )
+
+
+class AliceAgent(Agent):
+    """
+    Agent returns the Alice AIML bot's reply to an observation.
+
+    This is a strong rule-based baseline.
+    """
+
+    def __init__(self, opt, shared=None):
+        """
+        Initialize this agent.
+        """
+        super().__init__(opt)
+        self.id = 'Alice'
+        self.kern = None
+        self.load_alice()
+
+    def load_alice(self):
+        self.kern = aiml.Kernel()
+        self.kern.verbose(False)
+        self.kern.setTextEncoding(None)
+        chdir = os.path.join(aiml.__path__[0], 'botdata', 'alice')
+        self.kern.bootstrap(
+            learnFiles="startup.xml", commands="load alice", chdir=chdir
```

Comment:
will this load everything? ALICE has a *ton* of responses afaik.

Reply:
Yeah, it'll load all the `.aiml` files in the `alice` folder from `python-aiml`. It only takes me about 2 seconds to load all of it.

Reply:
Can you write a test  which just tries to run it on -t convai2, and put it in the nightly cpu folder?

Reply:
Should this also check that f1 or accuracy is at some value or just that it runs?

Reply:
Let's check f1. I want to primarily ensure that it is stable over time.